src/coreclr/jit/gentree.cpp

```diff
@@ -5099,6 +5099,13 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
                 costSz = 7; // jump to cold section
                 break;
 
+#ifdef TARGET_ARM64
+            case GT_CHK_DIV_BY_ZERO:
+                costEx = 4; // cmp reg,reg and jae throw (not taken)
```

Comment:
not sure what the values should be, right now its just a copy of GT_BOUNDS_CHECK